src/Configuration/Schema/SchemaConfiguration.php

```diff
@@ -52,13 +52,24 @@ final class SchemaConfiguration
     private $logs;
     private $tmpDir;
     private $phpUnit;
+    private $ignoreMsiWithNoMutations;
+    /**
+     * @var float|int
+     */
+    private $minMsi;
+    /**
+     * @var float|int
+     */
+    private $minCoveredMsi;
     private $mutators;
     private $testFramework;
     private $bootstrap;
     private $initialTestsPhpOptions;
     private $testFrameworkExtraOptions;
 
     /**
+     * @param float|int $minMsi
```

Comment:
what about just `float` and add type declaration?

Reply:
👍